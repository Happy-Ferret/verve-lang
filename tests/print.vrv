print(1)
print(1.5)
print([1, 5])
print(["Hello",  "World"])
print("Hello, World")
