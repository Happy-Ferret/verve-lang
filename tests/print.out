1
1.5
[1, 5]
[Hello, World]
Hello, World
